woodwork/logical_types.py

```diff
@@ -15,6 +15,7 @@
 from woodwork.utils import (
     _infer_datetime_format,
     _is_valid_latlong_series,
+    _is_valid_latlong_value,
```

Comment:
NAB , is this an anti-pattern? Importing a "private" method?

Reply:
Yes, I believe so. It happens a lot throughout the codebase.